src/packages/core/src/options/server-options.ts

```diff
@@ -63,20 +63,24 @@ export const ServerOptions: Definitions<ServerConfig> = {
     normalize,
     shortDescription: "Enable a websocket server.",
     default: () => true,
-    legacyName: "ws"
+    legacyName: "ws",
+    cliType: "boolean",
+    cliAliases: []
```

Comment:
The default for aliases after merging #721 will be `[]` (https://github.com/trufflesuite/ganache-core/pull/721/files#diff-2b2281ab4da879ef24d594d2dc8cbee929624f9b3e1438f88d20694aa0351490R94). So I think it is safe to remove this.